app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -618,16 +719,23 @@ $after-button-padding-left: govuk-spacing(4);
   }
 }
 
+.gem-c-layout-super-navigation-header__search-toggle-button--large-navbar {
+  height: $large-navbar-height;
+}
+
+// to stop the search icon moving on hover
```

Comment:
I'm still seeing a slight "jump" when I hover over the search icon and the close X icon. This is on Chrome desktop 117.0.5938.132 and Mac.